### PR TITLE
Made changes to the new branch

### PR DESCRIPTION
**Pull request by TayyabKM**
Modified the HTML file, CSS file, and README file to add the contact form to the mobile version of the website.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -237,10 +238,18 @@ <h3><span></span>Skills</h3>
...
+              <form action="https://formspree.io/f/xqkjbgbd"  method="post">
+                <ul>
+                  <li>
+                    <input type="text" id="fname" name="fname" placeholder="Full name" maxlength="30" required>
+                  </li>
+                  <li>
+                    <input type="email" id="email" name="email" placeholder="Email address" required>
+                  </li>
+                  <li>
+                    <textarea name="textarea" id="textarea" rows="10" placeholder="Enter text here" maxlength="500" required></textarea>
```
Comment (SSEKPIUS) [inline index.html:236]: this is an awesome addition,  @TayyabKM  thanks for the good work

**Review by SSEKPIUS — APPROVED** [review]
we are ready to merge, thanks for good work

File: index.html (hunk 1 above)
Comment (roshan-bajgain) [inline index.html:236]: - [ ] You should stick to the design templates in Figma as much as possible, So I would suggest you please remove the  border and provide padding for placeholder
### Fimga design
fimga
### Your design
his

File: index.html [hunk 2]
```diff
@@ -237,10 +238,18 @@ <h3><span></span>Skills</h3>
...
+                  </li>
+                </ul>
                 <input type="submit" value="Get in touch">
               </form>
```
Comment (roshan-bajgain) [inline index.html:240]: - [ ] You should stick to the design templates in Figma as much as possible, So please remove the black border So it matches the design
### Your design
his
### Fimga design
figma

File: index.html (hunk 1 above)
Comment (roshan-bajgain) [inline index.html:236]: - [ ] I would suggest you please, implement button interactions such as hover, active (when it's pressed), disabled, and enable (default state) styled as shown in the Figma template.
### Figma template
gihms

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @SSEKPIUS, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jerryowusu — APPROVED** [review]
Hi @TayyabKM :wave: 
 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

weldone



## Highlights
:heavy_check_mark: All linters are passing
:heavy_check_mark: Nice readme file
:heavy_check_mark: Excellent commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**